apps/hash-frontend/src/pages/entities.page.tsx

```diff
@@ -0,0 +1,220 @@
+import { getLayoutWithSidebar, NextPageWithLayout } from "../shared/layout";
+import { HEADER_HEIGHT } from "../shared/layout/layout-with-header/page-header";
+import { tableHeaderHeight } from "../shared/table-header";
+import { Button } from "../shared/ui";
+import { TabLink } from "../shared/ui/tab-link";
+import { Tabs } from "../shared/ui/tabs";
+import { EntitiesTable } from "./shared/entities-table";
+import {
+  TOP_CONTEXT_BAR_HEIGHT,
+  TopContextBar,
+} from "./shared/top-context-bar";
+import { WorkspaceContext } from "./shared/workspace-context";
+
+const contentMaxWidth = 1000;
+
+type ParsedQueryParams = {
+  entityTypeIdOrBaseUrl?: string;
+};
+
+const EntitiesPage: NextPageWithLayout = () => {
+  const router = useRouter();
+  const { activeWorkspace } = useContext(WorkspaceContext);
+
+  const { entityTypeId, entityTypeBaseUrl } = useMemo(() => {
+    if (router.isReady) {
+      const { entityTypeIdOrBaseUrl } = router.query as ParsedQueryParams;
+
+      return {
+        entityTypeId:
+          entityTypeIdOrBaseUrl && !isBaseUrl(entityTypeIdOrBaseUrl)
+            ? (entityTypeIdOrBaseUrl as VersionedUrl)
+            : undefined,
+        entityTypeBaseUrl:
+          entityTypeIdOrBaseUrl && isBaseUrl(entityTypeIdOrBaseUrl)
+            ? entityTypeIdOrBaseUrl
+            : undefined,
+      };
+    }
+    return {};
+  }, [router]);
+
+  const { lastRootPageIndex } = useAccountPages(
+    activeWorkspace?.accountId as OwnedById,
+  );
+  const [createUntitledPage] = useCreatePage(
+    activeWorkspace?.accountId as OwnedById,
+  );
+
+  const createPage = useCallback(async () => {
+    await createUntitledPage(lastRootPageIndex);
+  }, [lastRootPageIndex, createUntitledPage]);
+
+  const latestEntityTypes = useLatestEntityTypesOptional({
+    includeArchived: true,
+  });
+
+  const entityType = useMemo(
+    () =>
+      entityTypeId || entityTypeBaseUrl
+        ? latestEntityTypes?.find(({ schema }) =>
+            entityTypeId
+              ? schema.$id === entityTypeId
+              : extractBaseUrl(schema.$id) === entityTypeBaseUrl,
+          )
+        : undefined,
+    [latestEntityTypes, entityTypeId, entityTypeBaseUrl],
+  );
+
+  const entityTypeEntitiesValue = useEntityTypeEntitiesContextValue({
+    entityTypeBaseUrl,
+    entityTypeId,
+  });
+
+  const isViewAllPagesPage =
+    entityType?.schema.$id === types.entityType.page.entityTypeId;
+
+  const pageTitle = entityType ? `${entityType.schema.title}s` : "Entities";
+
+  const theme = useTheme();
+
+  const { entities, loading } = entityTypeEntitiesValue;
+
+  return (
+    <>
+      <NextSeo title={pageTitle} />
+      <TopContextBar
+        defaultCrumbIcon={null}
+        crumbs={[
+          {
+            title: pageTitle,
+            href: isViewAllPagesPage
+              ? "/pages"
+              : `/entities${
+                  entityTypeId ? `?entityTypeId=${entityTypeId}` : ""
```

Comment:
This won't work because `entityTypeId` is not a query param we're checking for at the moment

Reply:
Thanks for catching, have addressed in https://github.com/hashintel/hash/pull/2924/commits/b6aaf595bf7bd1e7252272a4299a0f206ef23b6c